libs/build_esma_cmake.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+set -eux
+
+name="esma_cmake"
+repo=${1:-${STACK_esma_cmake_repo:-"GEOS-ESM"}}
+version=${2:-${STACK_esma_cmake_version:-"main"}}
+id=${version//\//-}
+
+if $MODULES; then
+  set +x
+  source $MODULESHOME/init/bash
+  module list
+  set -x
+
+  prefix="${PREFIX:-"/opt/modules"}/core/$name/$repo-$id"
```

Comment:
should be just `$id`

Reply:
Done in 81e539f4a6443c4253b9e4825eeeebcd28bda884